faq/faq.rst

```diff
@@ -177,10 +177,17 @@ As the error message suggests, you can (cautiously) increase this limit by alter
 
   SET citus.max_intermediate_result_size = '3GB';
 
-Can I run Citus on Heroku or Amazon RDS?
-----------------------------------------
+Can I run Citus on Heroku?
+--------------------------
+
+Yes, the `Citus Heroku add-on <https://elements.heroku.com/addons/citus>`_ provisions a Citus cluster and makes it available to a Heroku app.
+
+Can I run Citus on Amazon RDS?
+------------------------------
+
+At this time Amazon does not support running Citus directly on top of Amazon RDS. It is up to them if/when they enable the Citus extension.
 
-At this time Heroku and Amazon do not support running Citus directly on top of Heroku PostgreSQL or Amazon RDS. It is up to them if/when they enable the Citus extension. If you are looking for something similar, `Citus Cloud <https://www.citusdata.com/product/cloud>`_ is our database-as-a-service which we fully manage for you. It runs on top of AWS (like both RDS and Heroku PostgreSQL) and should provide a very similar product experience, with the addition of Citus' horizontal scaling.
+If you are looking for something similar, `Citus Cloud <https://www.citusdata.com/product/cloud>`_ is our database-as-a-service which we fully manage for you. It runs on top of AWS (like both RDS and Heroku PostgreSQL) and should provide a very similar product experience, with the addition of Citus' horizontal scaling. You can even provision Citus Cloud with your existing AWS account using `Citus Cloud on the AWS Marketplace <https://aws.amazon.com/marketplace/pp/B076M3F79H>`_.
```

Comment:
For Marketplace, can we add a separate FAQ item? E.g., "Can I use Citus with my existing AWS account"? (or some such), and then provide the Marketplace answer?